src/api/rosetta-constants.ts

```diff
@@ -12,8 +12,11 @@ export const RosettaConstants = {
   rosettaVersion: '1.4.6',
   symbol: 'STX',
   decimals: 6,
-  lockedBalance: 'LockedBalance',
-  spendableBalance: 'SpendableBalance',
+  StakedBalance: 'StakedBalance',
```

Comment:
Should this be `Stacked` rather than `Staked`?

Reply:
yeah, fixed it.